src/puppetlabs/puppetdb/http/fact_names.clj

```diff
@@ -25,7 +25,8 @@
                  version
                  (http-q/validate-distinct-options! (merge (keywordize-keys params) puppetdb-query))
                  (:scf-read-db globals)
-                 (:url-prefix globals))))
+                 (:url-prefix globals)
+                 (:pretty-print globals))))
```

Comment:
sure, but that's not user-facing. I mean it may be worth stating something explicit with respect to fact-names and the pretty_print param